### DISCUSSION TITLE
Unix time counter (seconds since an epoch) in Zephyr

### DISCUSSION THREAD

**Discussion by GrixaYrev** [zephyrproject-rtos/zephyr#51253] [Q&A] [answered]
Is there unix time counter in Zephyr?
I mean a counter which have API like this:
``` C
void set_unix_time(time_t);
time_t get_unix_time(void);
```
And incremeanting is encapsulated.
Or i need implement it in my project?

**Comment by yashi** [comment, marked as answer]
There are
- `clock_gettime()`
- `timer_gettime()`

under the [POSIX API](https://docs.zephyrproject.org/latest/doxygen/html/include_2zephyr_2posix_2time_8h.html).
